sdk/src/main/java/com/ingresse/sdk/request/Highlight.kt

```diff
@@ -15,6 +17,7 @@ interface Highlight {
     @GET("/featured")
     fun getHighlightEvents(@Query("apikey") apikey: String,
                            @Query("state") state: String? = null,
+                           @Query("method") method: HighlightMethod? = null,
```

Comment:
Por que aqui você já não passou direto "banner" como method??

Reply:
Adicionei um comentário na próxima.

Reply:
Lembrando que o SDK é desenvolvido pensando no uso externo, logo, para quem for integrar, deve ao máximo conhecer as possibilidades do mesmo e então realizar as escolhas de defaults a parte.